#! /bin/sh -e

# All lines beginning with `# DP:' are a description of the patch.
# DP: Description: Make glibc-2.3.5 compile with gcc-4.0 on ia64 fixed by
#		   removing strong_alias.
# DP: Related bugs: 
# DP: Dpatch author: GOTO Masanori <gotom@debian.org>
# DP: Patch author: GOTO Masanori <gotom@debian.org>
# DP: Upstream status: In CVS
# DP: Status Details: 
# DP: Date: 2005-07-27

2005-07-16  GOTO Masanori  <gotom@debian.org>

	* sysdeps/alpha/dl-machine.h: Remove strong_alias.

--- sysdeps/alpha/dl-machine.h.gotom	2005-07-28 14:14:51.803668403 +0900
+++ sysdeps/alpha/dl-machine.h	2005-07-28 14:15:11.440036959 +0900
@@ -289,8 +289,7 @@ elf_machine_runtime_setup (struct link_m
   TRAMPOLINE_TEMPLATE (_dl_runtime_profile, profile_fixup, /* nop */);
 #else
 #define ELF_MACHINE_RUNTIME_TRAMPOLINE				\
-  TRAMPOLINE_TEMPLATE (_dl_runtime_resolve, fixup, imb);	\
-  strong_alias (_dl_runtime_resolve, _dl_runtime_profile);
+  TRAMPOLINE_TEMPLATE (_dl_runtime_resolve, fixup, imb);
 #endif
 
 /* Initial entry point code for the dynamic linker.
